karaf/features/src/main/feature/feature.xml

```diff
@@ -55,7 +55,12 @@
         <bundle>mvn:javax.annotation/javax.annotation-api/${cxf.javax.annotation-api.version}</bundle>
     </feature>
 
+    <feature name="brooklyn-config" version="${project.version}">
+        <feature>brooklyn-server-config</feature>
+    </feature>
+
     <feature name="brooklyn-headless" version="${project.version}" description="All Brooklyn bundles witht the exception of the launcher">
+        <feature prerequisite="true">brooklyn-config</feature>
```

Comment:
Gut feel is that we should move this line to after the feature `brooklyn-guava-optional-deps`. I think that the "prerequisite" features are resolved/loaded in-order. We presumably want all of the standard karaf stuff in place before we try installing/starting the `brooklyn-server-config` feature.